pkg/resources/dynamic_table_acceptance_test.go

```diff
@@ -87,6 +103,58 @@ func TestAcc_DynamicTable_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "target_lag.#", "1"),
 					resource.TestCheckResourceAttr(resourceName, "target_lag.0.downstream", "true"),
 					resource.TestCheckResourceAttr(resourceName, "comment", "Terraform acceptance test - updated"),
+
+					resource.TestCheckResourceAttrWith(resourceName, "created_on", func(value string) error {
+						if value != createdOn {
+							return fmt.Errorf("created_on changed from %v to %v", createdOn, value)
+						}
+						return nil
+					}),
+				),
+			},
+			// test changing initialize setting
+			{
+				ConfigDirectory: config.TestStepDirectory(),
+				ConfigVariables: variableSet3,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+					resource.TestCheckResourceAttr(resourceName, "database", acc.TestDatabaseName),
+					resource.TestCheckResourceAttr(resourceName, "schema", acc.TestSchemaName),
+					resource.TestCheckResourceAttr(resourceName, "initialize", "ON_SCHEDULE"),
+					resource.TestCheckResourceAttr(resourceName, "refresh_mode", "AUTO"),
+					resource.TestCheckResourceAttr(resourceName, "target_lag.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "target_lag.0.downstream", "true"),
+					resource.TestCheckResourceAttr(resourceName, "comment", "Terraform acceptance test"),
+
+					resource.TestCheckResourceAttrWith(resourceName, "created_on", func(value string) error {
+						if value == createdOn {
+							return fmt.Errorf("expected created_on to change but was not changed")
+						}
+						createdOn = value
+						return nil
+					}),
+				),
+			},
+			// test changing refresh_mode setting
+			{
+				ConfigDirectory: config.TestStepDirectory(),
+				ConfigVariables: variableSet4,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+					resource.TestCheckResourceAttr(resourceName, "database", acc.TestDatabaseName),
```

Comment:
a cosmetic change for me would be not to validate every single param with every step, but only the ones that we are changing but this is not required to merge this PR